### ISSUE TITLE
New implementation of path/railing separation breaks old parks

### ISSUE THREAD

**Issue by ArcticX9** [OpenRCT2/OpenRCT2#19503]
### Operating System

Windows 10

### OpenRCT2 build

OpenRCT2, v0.4.3-313-gOda9693

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error, This issue has to do with park objects and/or asset packs

### Describe the issue

Recently I have noticed that, since the implementation of selectable/mix-and-matchable path types and railings, a Tile Inspector trick used in the past featuring queue path being made invisible yet maintaining path objects has become broken.

The trick involved used tile inspector to select three out of four directions, making the queue line invisible but maintaining the path features.
![Old Behavior](https://user-images.githubusercontent.com/20503645/221441857-8673a34a-2d22-4f12-9d81-752fc1a59db8.png)

When I opened a project recently, I noticed this behavior had changed.
![New Behavior](https://user-images.githubusercontent.com/20503645/221441884-0630ef21-d3d0-489f-acbd-e89d5f47f852.png)

Upon investigating older parks, I found that those parks (many which were pre-.park format) had developed this issue retroactively. It's even visible in the official OpenRCT2 title sequence (Ancient Worlds).
![Opening](https://user-images.githubusercontent.com/20503645/221441922-6251000e-ab3b-4474-97b8-6a0cdb18dac0.png)
![AW New 4](https://user-images.githubusercontent.com/20503645/221441925-278ad609-f6df-456e-b9a6-59d19c367ad5.png)

### Steps to reproduce

1. Open any of the attached parks or any park that had used this method in the past.
2. Find an instance of the technique used; in the case of Ancient Worlds this is clearly visible at the entrance and the boat docks near the entrance.
3. Note that there is now queue railings where there once was nothing.

Old:
![AW Old](https://user-images.githubusercontent.com/20503645/221441980-1c4fc38d-cfc3-4781-9968-436168f59750.png)
New:
![AW New 1](https://user-images.githubusercontent.com/20503645/221441983-dbcb7aa1-732d-4347-81a0-7b95518e78a6.png)
![AW New 2](https://user-images.githubusercontent.com/20503645/221441985-81a33853-14be-4616-8249-02a5c3bba15d.png)

### Attachments

![AW Old](https://user-images.githubusercontent.com/20503645/221441472-231675b1-66ca-4cdb-b9eb-c1947d7e818a.png)
![AW New 1](https://user-images.githubusercontent.com/20503645/221441474-17fa9dc6-0afc-4cf3-a6b9-8f64a8e22d50.png)
![AW New 2](https://user-images.githubusercontent.com/20503645/221441476-ce2bfc51-3848-4e7b-a78f-45ea61bba31b.png)
![AW New 3](https://user-images.githubusercontent.com/20503645/221441480-a0987192-b9c2-49a0-8c2d-ece59621f380.png)
![HH New](https://user-images.githubusercontent.com/20503645/221441481-8beddc70-9a64-440c-b553-ff3b6455c646.png)
![Marblehead New](https://user-images.githubusercontent.com/20503645/221441484-3b5844e6-8f67-4d67-9036-e822fa8cabdd.png)
![EG New 3](https://user-images.githubusercontent.com/20503645/221441486-358
![SG New](https://user-images.githubusercontent.com/20503645/221441488-f9d2aa8e-374a-4728-8d84-5ae3ff7f7ff9.png)
3945e-201f-4e92-b3d4-f3f1780c8d0e.png)
![AW New 4](https://user-images.githubusercontent.com/20503645/221441772-8d202bfd-f10a-4f2c-bfcb-8e49c3f5c119.png)
![Opening](https://user-images.githubusercontent.com/20503645/221441848-a03a9dbd-e060-4b3a-91ff-1655f5918d90.png)

[References.zip](https://github.com/OpenRCT2/OpenRCT2/files/10834910/References.zip)

**Comment by spacek531** [comment]
The queue railings are likely caused by an update that allowed queues to have forks in them. The solution would be to import queue paths like this as invisible from sv6 and older park files (there could be some issues if park file version was not bumped).

**Comment by Gymnasiast** [comment]
Yes, this wouldn't be difficult to fix. I actually employed a solution like that for the steep log flume bits, so it shouldn't be difficult to port. Do note that SV4 would also need to be modified, as it's possible the same trick was used there.

**Comment by spacek531** [comment]
I'll take this bug.

**Comment by spacek531** [comment]
will be fixed by https://github.com/OpenRCT2/OpenRCT2/pull/19538
